ts/session/constants.ts

```diff
@@ -45,7 +45,7 @@ export const CONVERSATION = {
 export const MAX_ATTACHMENT_FILESIZE_BYTES = 6 * 1000 * 1000; // 6MB
 
 export const VALIDATION = {
-  MAX_GROUP_NAME_LENGTH: 64,
+  MAX_GROUP_NAME_LENGTH: 30,
```

Comment:
Should we make all max name lengths the same value? For example `MAX_USERNAME_LENGTH` could also be 30 characters.

Reply:
That should probably be part of another task. I've fixed that one as it was part of QA for the closed group input, but I am not sure if the limit of 30 is applied to every other platforms for the MAX_USERNAME_LENGTH

Reply:
Cool 👍